Missing check for pcl_conversions library

On Debian unstable, installation of the `libpcl-dev` package lets the cmake configure step pass successfully. But during compilation I get:

    [ 58%] Building CXX object cartographer_ros/CMakeFiles/msg_conversion.dir/msg_conversion.cc.o
    In file included from /home/zft/cartographer/cartographer_ros/cartographer_ros/cartographer_ros/msg_conversion.cc:17:0:
    /home/zft/cartographer/cartographer_ros/cartographer_ros/cartographer_ros/msg_conversion.h:30:45: fatal error: pcl_conversions/pcl_conversions.h: No such file or directory
     #include "pcl_conversions/pcl_conversions.h"
                                                 ^
    compilation terminated.
    cartographer_ros/CMakeFiles/msg_conversion.dir/build.make:62: recipe for target 'cartographer_ros/CMakeFiles/msg_conversion.dir/msg_conversion.cc.o' failed
    make[2]: *** [cartographer_ros/CMakeFiles/msg_conversion.dir/msg_conversion.cc.o] Error 1
    CMakeFiles/Makefile2:1754: recipe for target 'cartographer_ros/CMakeFiles/msg_conversion.dir/all' failed
    make[1]: *** [cartographer_ros/CMakeFiles/msg_conversion.dir/all] Error 2
    Makefile:138: recipe for target 'all' failed
    make: *** [all] Error 2

This error is solved after installing the `libpcl-conversions-dev` package but the check for `pcl_conversions.h` should be done during the configuration step.

--- a/cartographer_ros/CMakeLists.txt
+++ b/cartographer_ros/CMakeLists.txt
@@ -17,12 +17,13 @@ cmake_minimum_required(VERSION 2.8)
 project(cartographer_ros)
 
 set(PACKAGE_DEPENDENCIES
+  cartographer_ros_msgs
   eigen_conversions
   geometry_msgs
-  cartographer_ros_msgs
+  nav_msgs
+  pcl_conversions
   roscpp
   sensor_msgs
-  nav_msgs
   tf2
   tf2_eigen
   tf2_ros

--- a/cartographer_ros/package.xml
+++ b/cartographer_ros/package.xml
@@ -32,16 +32,22 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>g++-static</build_depend>
   <build_depend>google-mock</build_depend>
-  <build_depend>python-sphinx</build_depend>
+  <build_depend>g++-static</build_depend>
   <build_depend>protobuf-dev</build_depend>
+  <build_depend>python-sphinx</build_depend>
 
   <depend>cartographer</depend>
   <depend>cartographer_ros_msgs</depend>
 
+  <depend>eigen_conversions</depend>
+  <depend>geometry_msgs</depend>
   <depend>libgflags-dev</depend>
   <depend>libgoogle-glog-dev</depend>
+  <depend>libpcl-all-dev</depend>
+  <depend>message_runtime</depend>
+  <depend>nav_msgs</depend>
+  <depend>pcl_conversions</depend>
   <depend>rosbag</depend>
   <depend>roscpp</depend>
   <depend>roslib</depend>
@@ -50,12 +56,6 @@
   <depend>tf2</depend>
   <depend>tf2_eigen</depend>
   <depend>tf2_ros</depend>
-  <depend>geometry_msgs</depend>
-  <depend>nav_msgs</depend>
-  <depend>libpcl-all-dev</depend>
-  <depend>pcl_conversions</depend>
-  <depend>eigen_conversions</depend>
-  <depend>message_runtime</depend>
   <depend>yaml-cpp</depend>
 
   <test_depend>rosunit</test_depend>

